README.mdx

```diff
@@ -0,0 +1,206 @@
+# TODO: Expand on outline
+
+> :warning: This guidance is in prep for a Playground tutorial, but attachments must first be enabled on Playground for
+this guidance to make sense.
+
+# KittyVerse + KittyHats
+
+To showcase how attachments unlock permissionless composability on Flow, this tutorial will walk you though making your
+own cat-inspired NFT collection and a wardrobe of hats to go along with them. Let's put some hats on some cats!
+
+## What is composability?
+
+Before digging in, let's talk about what we mean by **_composability_**.
+
+<Callout type="info">
+
+Composability is the ability for existing components in one construction to be reused and even extended for another 
+purpose. <br/>
+
+Think of composable architectures like [building with legos](https://twitter.com/cdixon/status/1450462593232089088?s=20) 
+- you build something you need as a block or set of blocks and someone else can come along and use those same blocks to 
+build something entirely new.<br/>
+
+Composability on Flow, especially with attachments, is permissionless because not only can you build with existing 
+components, but no one can stop you from doing so! Hence the term permissionless!
+
+</Callout>
+
+Permissionless composability is one of the unique characteristics of decentralized app development because, as builders,
+we can freely compose richer or derivative functionalities on top of any existing contracts from other applications or
+communities. Contracts that you deploy which interact with other contracts are not subject to internal changes or
+control by the contract you composed it upon. 
+
+Cadence is uniquely suited for composable architectures because Cadence contracts can compose other contracts. As a
```

Comment:
I don't know what is the right way to put it but this line seems odd to me ` because Cadence contracts can compose other contracts`.  Maybe re-write like this `Cadence excels in facilitating composable architectures due to its ability to compose other contracts through Cadence contracts.`